W3ChampionsIdentificationService/RolesAndPermissions/Controllers/PermissionsController.cs

```diff
@@ -64,5 +65,12 @@ public async Task<IActionResult> Update([FromBody] Permission permission)
             }
             return Ok();
         }
+        [HttpPost("checkPermission")]
+        [B2BVerification]

```

Comment:
If we're adding this functionality, we should also add it to more endpoints (if that's to come later, no problem)

Reply:
yeah, I was planing to add later, it's just MVP to introduce new approach